### PR TITLE
Simplify example with updated sbt-scalahost

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,43 +1,14 @@
-// Scala.meta supports two most recent major versions of Scala.
-// Scalahost, being a compiler plugin, only supports the most recent minor versions.
-lazy val ScalaVersion = ScalaVersions.head
-lazy val ScalaVersions = Seq("2.11.8", "2.12.1")
+inThisBuild(List(
+  scalaVersion := "2.11.8" // 2.12 is also supported
```
Comment (xeno-by) [inline build.sbt]: `s/2.12/2.12.1/`

Comment (olafurpg) [reply build.sbt]: Done. Force-pushed.

**Comment by olafurpg** [comment]
We'll need to update the version number once 1.6 is out.

**Comment by xeno-by** [comment]
@olafurpg Yep, already have a reminder to do that.

**Pull request merged**